specification/web/resource-manager/Microsoft.Web/stable/2016-06-01/examples/managedApisGet.json

```diff
@@ -0,0 +1,65 @@
+{
+  "parameters": {
+    "subscriptionId": "34adfa4f-cedf-4dc0-ba29-b6d1a69ab345",
+    "location": "northcentralus",
+    "apiName": "testManagedApi",
+    "api-version": "2016-06-01"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "properties": {
+          "name": "testManagedApi",
+          "connectionParameters": {
+            "token": {
+              "type": "oauthSetting",
+              "oAuthSettings": {
+                "identityProvider": "oauth2",
+                "clientId": "totallyrealclientid",
+                "scopes": [
+                  "read",
+                  "write"
+                ],
+                "redirectUrl": "https://logic-apis-northcentralus.consent.azure-apim.net/redirect",
+                "properties": {
+                  "IsFirstParty": "False"
+                },
+                "customParameters": {
+                  "authorizationUrl": {
+                    "value": "https://10to8.com/o/authorize/"
```

Comment:
https://10to8.com is a real service. I'm not sure if we should include third party URLs in any sample, except for well-known public test services.

Reply:
I agree, will change to tempuri.org as that is the pattern I was using elsewhere